packages/next/export/worker.js

```diff
@@ -73,14 +74,18 @@ process.on(
           ...headerMocks
         }
 
+        if (sprPage && isDynamicRoute(page)) {
```

Comment:
Also, we're discussing to make this strictly `if (sprPage) { ... }`. Just something to keep in mind.